recipes/pango/all/conanfile.py

```diff
@@ -98,47 +117,55 @@ def package_info(self):
         self.cpp_info.components['pango_'].requires.append('glib::gio-2.0')
         self.cpp_info.components['pango_'].requires.append('fribidi::fribidi')
         self.cpp_info.components['pango_'].requires.append('harfbuzz::harfbuzz')
-        if self.settings.os != "Windows":
+        if self.options.with_fontconfig:
             self.cpp_info.components['pango_'].requires.append('fontconfig::fontconfig')
-            if self.settings.os == "Linux":
+
+
+        if self.options.with_xft:
+            self.cpp_info.components['pango_'].requires.append('xorg::xft')
+            # Pango only uses xrender when Xft, fontconfig and freetype are enabled
+            if self.options.with_fontconfig and self.options.with_freetype:
                 self.cpp_info.components['pango_'].requires.append('xorg::xrender')
-                self.cpp_info.components['pango_'].requires.append('xorg::xft')
-        self.cpp_info.components['pango_'].requires.append('cairo::cairo_')
+        if self.options.with_cairo:
+            self.cpp_info.components['pango_'].requires.append('cairo::cairo_')
         self.cpp_info.components['pango_'].includedirs = [os.path.join(self.package_folder, "include", "pango-1.0")]
         
-        if self.settings.os != "Windows":
+        if self.options.with_freetype:
```

Comment:
```suggestion
        if self.options.with_freetype and self.options.with_fontconfig:
```
see https://gitlab.gnome.org/GNOME/pango/-/blob/1.48.0/meson.build#L358